exercises/crypto-square/crypto_square.py

```diff
@@ -1,2 +1,2 @@
-def encode():
+def encode(string):
```

Comment:
I believe `text` or `message` would be a better option since `string` is a very general term.
What do you think?